api/v2/Routes/students/progressquerystring/index.js

```diff
@@ -42,7 +51,18 @@ router.get('/', async (req, res) => {
             userTopicPoints[topic] = Math.ceil(unBoundedMasteryLevel);
         }
     });
-    return res.status(200).json(Object.values(userTopicPoints).join(''));
+    let masteryNum = Object.values(userTopicPoints).join('');
+    return masteryNum;
+}
+
+router.get('/', async (req, res) => {
+    const email = res.locals.email
```

Comment:
let's use the other approach where we get it from the query params

Reply:
eg: https://github.com/Connor-Bernard/gradeView/pull/143/files#diff-b871c2476326f00dbf6332d98dc581eda9b7450d9f9208bb0d4e5071339fdd16L20